src/main/java/com/samsolutions/kitayeu/myproject/converters/UserToDtoConverter.java

```diff
@@ -0,0 +1,18 @@
+package com.samsolutions.kitayeu.myproject.converters;
+
+import com.samsolutions.kitayeu.myproject.dtos.UserDto;
+import com.samsolutions.kitayeu.myproject.entities.User;
+import lombok.NonNull;
+import org.springframework.beans.BeanUtils;
+import org.springframework.core.convert.converter.Converter;
+
+public class UserToDtoConverter implements Converter<User, UserDto> {
+
+    @Override
+    public UserDto convert(@NonNull User user) {
```

Comment:
@NonNull is lombok annotation. @NotNull - is validation. But both do not work here